lm_eval/evaluator.py

```diff
@@ -72,6 +76,11 @@ def simple_evaluate(
         lm = lm_eval.models.get_model(model).create_from_arg_string(
             model_args, {"batch_size": batch_size, "max_batch_size": max_batch_size, "device": device}
         )
+    elif isinstance(model, transformers.PreTrainedModel):
+        lm = HFLM(
+                pretrained=model,
```

Comment:
We also want to pass `batch_size=batch_size` to this I believe. Agree that we should assume the user has already placed their model onto the correct device though!

Reply:
`lm = lm_eval.models.get_model("hf-causal")` instead of instantiating HFLM directly here too preferably.